SoftLayer/CLI/image/detail.py

```diff
@@ -85,3 +87,51 @@ def _get_transaction_groups(children_images):
         transactions.add(utils.lookup(child, 'transaction', 'transactionGroup', 'name'))
 
     return transactions
+
+
+def _get_virtual_disks(children_images):
+    """Returns datacenters as a table.
+
+      :param children_images: A list of images.
+      """
+
+    table_virtual_disks = formatting.Table(['Device', 'capacity', 'size on disk'])
+
+    if utils.lookup(children_images[0], 'blockDevices'):
```

Comment:
Add a check to make sure children_images actually has things in it.

Reply:
The function `lookup` check if the attribute exist